flutter_app/lib/main.dart

```diff
@@ -35,44 +36,15 @@ class HomePage extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(title),
+        title: Center(child: Text(title)),
       ),
-      body: Column(
-        children: [
-          // attitude
-          DataField<double>(
-            name: 'Yaw (deg)',
-            value: comm.getYawStream(),
-            formatter: (double value) => (value / pi * 180.0).round(),
-          ),
-          DataField<double>(
-            name: 'Pitch (deg)',
-            value: comm.getPitchStream(),
-            formatter: (double value) => (value / pi * 180.0).round(),
-          ),
-          DataField<double>(
-            name: 'Roll (deg)',
-            value: comm.getRollStream(),
-            formatter: (double value) => (value / pi * 180.0).round(),
-          ),
-
-          // global position
-          DataField<int>(
-            name: 'Latitude',
-            value: comm.getLatStream(),
-            formatter: (int value) => (value / 1e7).round(),
-          ),
-          DataField<int>(
-            name: 'Longitude',
-            value: comm.getLonStream(),
-            formatter: (int value) => (value / 1e7).round(),
-          ),
-          DataField<int>(
-            name: 'Altitude (m)',
-            value: comm.getAltStream(),
-            formatter: (int value) => (value / 1e3).round(),
-          ),
-        ],
+      body: Align(
```

Comment:
I think you should remove the Align as it might limit the responsiveness (how it handles small screens) of the app. Don't worry about alignment for now as we'll work that out when putting all the widgets together.